sdk/python/foundation-models/system/evaluation/image-object-detection/fridge-eval-config.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
is eval-config mandatory (for my knowledge)?

Why iou_thresh and box_score are not part of this config.